sdk/js/src/relayer/relayer/helpers.ts

```diff
@@ -314,17 +277,23 @@ export function getWormholeRelayerLog(
       sequence: log.args[1].toString(),
       nonce: log.args[2].toString(),
       emitterAddress: tryNativeToHexString(log.args[0].toString(), "ethereum"),
+      payload: log.args[3],
```

Comment:
will this def exist?

Reply:
believe so - this is a field on every wormhole 'publish message' event